Src/DependencyCollector/Shared/Implementation/ClientServerDependencyTracker.cs

```diff
@@ -165,7 +165,7 @@ internal static void AddTupleForWebDependencies(WebRequest webRequest, Dependenc
 
             Tuple<DependencyTelemetry, bool> telemetryTuple = null;
 
-            if (DependencyTableStore.Instance.IsProfilerActivated || PretendProfilerIsAttached)
+            if (DependencyTableStore.Instance.IsDesktopHttpDiagnosticSourceActivated || DependencyTableStore.Instance.IsProfilerActivated || PretendProfilerIsAttached)
             {
```

Comment:
This does not look right. This means that, even when Profiler is not installed, tuple will be looked up in SQLRequestConditionalHolder - this is incorrect, as without Profiler SQL Dependencies should be stored in SqlRequestCacheHolder.

(More concerning here is  - did any functional tests for sql fail? if not there is a test gap)